src/tests/pyhbac-test.py

```diff
@@ -536,7 +548,8 @@ def testHbacErrorString(self):
     if not res.wasSuccessful():
         error |= 0x3
 
-    suite = unittest.TestLoader().loadTestsFromTestCase(PyHbacRequestElementTest)
+    suite = \
+        unittest.TestLoader().loadTestsFromTestCase(PyHbacRequestElementTest)
```

Comment:
that's not very nice; maybe due to backslash

Reply:
Can you give me a suggestion on a better looking approach?